csrc/executor.cpp

```diff
@@ -1868,14 +1999,8 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
     }
   }
 
-  std::vector<std::vector<std::byte>> arg_buffers;
-  {
-    FUSER_PERF_SCOPE("ExecutorRunFusion::GetArgsBuffers");
-    arg_buffers.reserve(kernel()->parameters().size());
-    for (auto v : kernel()->parameters()) {
-      arg_buffers.emplace_back(
-          getKernelArgument(expr_eval, v, kernel()->indexType()));
-    }
```

Comment:
As discussed interactively, this was created (allocated) and dropped (deallocated) every time we `runFusion`, before. Part of the speedup here is not doing so many dynamic allocations to create the same array.